api/src/main/webapp/WEB-INF/cron_default.yaml

```diff
@@ -4,24 +4,44 @@ cron:
   schedule: every 3 hours
   timezone: UTC
   target: api
-- description: 'Daily sync of compliance training status (via Moodle API)'
+- description: |
+    Daily sync of compliance training status (via Moodle API). This is just a backup to the primary
+    flow. This would only ever catch the following scenarios:
+    1. The user's compliance training expires. The time scale on this expiration is O(years),
+    so syncing nightly is entirely acceptable.
+
+    2. The user completes compliance and doesn't return to the dashboard. We'd progress them in the
+    background. This won't make much of a difference and would have the same effect as if they had
+    navigated back to the dashboard.
+
+    3. Somehow the user manages to "uncomplete" training in Moodle. I'm not sure this is possible,
+    but if so, it's certainly an edge case where nightly would be acceptale latency
   url: /v1/cron/bulkSyncComplianceTrainingStatus
   schedule: every 24 hours
   timezone: UTC
   target: api
-- description: 'Daily sync of eRA Commons linkage status (via FireCloud API)'
+- description: >
+    Daily sync of eRA Commons linkage status (via FireCloud API).  I expect this cron to never
+    detect anything. This is effectively checking for whether any changes happened in the
+    background between the Terra account and its linkage to eRA commons. During onboarding,
+    this won't matter since it's checked live from the dashboard. After onboarding, I don't even
+    know whether it's possible to "unlink", and if a user did this, whether that even matters for
+    our registration policy. Certainly it's good to check, but none of this is time sensitive.
+    The current 24h interval is fine.
   url: /v1/cron/bulkSyncEraCommonsStatus
   schedule: every 24 hours
   timezone: UTC
   target: api
-- description: 'Daily sync of 2FA set up status (via Google Directory Service)'
+- description: >
+    Update our cache of users' two-factor authentication settings on their GSuite accounts via
```

Comment:
nit: would probably say "database cache" to avoid ambiguity here. If I read this as-is I would probably assume this meant memcache or other ephemeral storage.